tests/CompositionObject/OptionUnitTest.php

```diff
@@ -2,7 +2,7 @@
 namespace Slack\Tests\Object;
```

Comment:
Just one more to update here, but everything else looks good!

Reply:
done